package.json

```diff
@@ -37,6 +37,7 @@
     "ember-cli-emblem": "0.8.2-beta.2",
     "ember-cli-eslint": "3.0.2",
     "ember-cli-foundation-6-sass": "0.0.18",
+    "ember-cli-gravatar": "3.8.0",
```

Comment:
Don't we need to [whitelist gravatar.com in the CSP](https://github.com/johnotander/ember-cli-gravatar#content-security-policy)?